scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -329,12 +439,23 @@ function Uninstall-SecurityDaemon {
         [Switch] $DeleteMobyDataRoot,
```

Comment:
Change the commandlet description on line 404 to:

```
By default this commandlet does not delete the config.yaml and the Moby Engine data root (for -ContainerOs 'Windows' installs),
so that you can re-install the daemon using `Install-SecurityDaemon -ExistingConfig`. To delete these as well, specify the
-DeleteConfig and -DeleteMobyDataRoot flags.
```

ie change `update the daemon` to `re-install the daemon`, since just updating does not require uninstalling now.

Reply:
Fixed.